src/ContainerInstance/examples/Remove-AzContainerInstanceSubnetServiceAssociationLink.md

```diff
@@ -0,0 +1,22 @@
+### Example 1: {{ Add title here }}
+```powershell
+{{ Add code here }}
+```
+
+```output
+{{ Add output here }}
+```
+
+{{ Add description here }}
+
```

Comment:
Please add example for this new command and refresh its doc

Reply:
Reached out to the SME to update this command as I do not own this feature.

Reply:
Please raise another PR to update example.